src/sentry/static/sentry/app/views/onboarding/configure/index.jsx

```diff
@@ -89,7 +88,6 @@ const Configure = createReactClass({
   },
 
   submit() {
-    HookStore.get('analytics:onboarding-complete').forEach(cb => cb());
```

Comment:
is marketing capturing this a different way?

Reply:
don't think anyone is using this